Sources/SWBTaskExecution/TaskActions/ClangCompileTaskAction.swift

```diff
@@ -304,6 +318,24 @@ public final class ClangCompileTaskAction: TaskAction, BuildValueValidatingTaskA
                     return lastResult ?? .failed
                 }
             }
+
+            if let moduleDependenciesContext, let traceFile, lastResult == .succeeded {
```

Comment:
If clang doesn't have the feature, traceFile will be nil here, so we'll skip checking altogether, even if the user requested `VALIDATE_MODULE_DEPENDENCIES`. The Swift version on the other hand emits an error if `VALIDATE_MODULE_DEPENDENCIES` is set but the toolchain doesn't have the required support, mainly so that the test can check for that when it's running against older toolchains.

Reply:
OK. I already made it a warning for the Clang part here.